sympy/physics/mechanics/rigidbody.py

```diff
@@ -350,8 +349,8 @@ def parallel_axis(self, point):
 
         """
         # circular import issue
-        from sympy.physics.mechanics.functions import inertia
-        a, b, c = self.masscenter.pos_from(point).to_matrix(self.frame)
-        I = self.mass * inertia(self.frame, b**2 + c**2, c**2 + a**2, a**2 +
-                                b**2, -a * b, -b * c, -a * c)
-        return self.central_inertia + I
+        from sympy.physics.mechanics.functions import inertia_of_point_mass
+        if frame is None:
+            frame = self.frame
+        return self.central_inertia.express(frame) + inertia_of_point_mass(
+            self.mass, self.masscenter.pos_from(point), frame)
```

Comment:
You can add two dyadics without needing to express it in a frame. Why do you need to explicitly expression the central inertia in this frame?

Reply:
I'll remove the `express`, there is indeed no good reason to do so.